test/server/browser-provider-test.js

```diff
@@ -384,5 +396,70 @@ describe('Browser provider', function () {
             });
         });
     });
+
+    describe('Remote provider', () => {
+        it('Should log an error if a browser window was not found', async () => {
+            const bc   = new BrowserConnectionMock();
```

Comment:
Please remove extra space left to the equality sign here and in the next expression. Same thing in the next test.